certified-connectors/DocuSignDemo/apiDefinition.swagger.json

```diff
@@ -493,6 +493,74 @@
         "x-ms-visibility": "advanced"
       }
     },
+    "/accounts/{accountId}/envelopes/{envelopeId}/documents/{documentId}/documentsDownload": {
+      "get": {
+        "summary": "Get documents from envelope",
+        "description": "Get documents from envelope",
+        "operationId": "GetDocumentsWithDocumentId",
+        "parameters": [
+          {
+            "name": "accountId",
+            "in": "path",
+            "description": "Account id",
+            "required": true,
+            "x-ms-summary": "Account",
+            "x-ms-test-value": "insert account id",
+            "x-ms-dynamic-values": {
+              "operationId": "GetLoginAccounts",
+              "value-collection": "loginAccounts",
+              "value-path": "accountIdGuid",
+              "value-title": "name"
+            },
+            "type": "string"
+          },
+          {
+            "name": "envelopeId",
+            "in": "path",
+            "description": "Envelope id",
+            "required": true,
+            "x-ms-summary": "Envelope",
+            "x-ms-test-value": "insert envelope id",
+            "type": "string"
+          },
+          {
+            "name": "documentId",
+            "in": "path",
+            "description": "Provide specific document ID or use a keyword from the dropdown",
+            "x-ms-summary": "Document id",
+            "required": true,
+            "x-ms-test-value": "insert document id",
+            "type": "string",
+            "enum": [
+              "Get ZIP archive with documents and COC",
```

Comment:
just show the options like you see in the api.
`combined
archive
certificate
portfolio
`
In the documentation we can provide details.